chargeio/Program.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;  
+using Microsoft.Extensions.Logging;
+using chargeio;
+
+public class Program
```

Comment:
Removed in 1fdc20b16f44857efcc06b3794a6e35a0d6d46ae

Definitely a leftover from project generation. I used it to test a few things early on but it's safe to remove it now.